config.yml

```diff
@@ -56,6 +56,96 @@ commands:
                 curl -s -X POST -H "Accept: application/json" "${BASE_URL}/workflow/${WORKFLOW_ID}/cancel?${TOKEN}"
               fi
             done
+  slack-notify-waiting-for-approval:
```

Comment:
Nit: can you add a newline above this to visually separate it from all the above?